cmds/core/sluinit/uinit_linux.go

```diff
@@ -33,6 +58,28 @@ func printStep(msg string) {
 	step++
 }
 
+// printStepDisabled prints a message for a disabled step.
+func printStepDisabled(msg string) {
+	slaunch.Debug("******** %s disabled in config ********", msg)
+}
+
+// checkPolicyLocationFlag checks the kernel cmdline for the policyLocationFlag
+// flag. It provides the location of the policy file on disk. If it isn't set,
+// an error is returned.
+//
+// The flag takes an argument formatted as `<block device id>:<path>`
+//
+//	e.g., sda1:/boot/securelaunch.policy
+//	e.g., 4qccd342-12zr-4e99-9ze7-1234cb1234c4:/securelaunch.policy
+func checkPolicyLocationFlag() (string, error) {
+	location, present := cmdline.Flag(policyLocationFlag)
+	if !present {
+		return "", fmt.Errorf("'%s' command line flag is not set", policyLocationFlag)
```

Comment:
For writing tests and such, it's useful to use errors that can be wrapped. 
You can make your own new error messages:
var ErrFlagNotSet = errors.New("command line flag is not set")
and then use it
return "", fmt.Errorf("%q:%w", policyLocationFlag, ErrFlagNotSet)

Also, if you are doing this
'%s'
you can use %q instead. 
Finally, option 2 is something like this:
return "", fmt.Errorf("%q:%v", policyLocationFlag, os.ErrNotExist)
but then that may not give you a message you like.